lib/jekyll-contentful-data-import/data_exporter.rb

```diff
@@ -17,9 +17,24 @@ def initialize(name, entries, config = {})
 
       def run
         setup_directory
+        puts config
+        export_fn = config['multiple_files'] ? export_by_content_type : export
+        export_fn
+      end
 
+      def export
+        grouped_entries = ::Jekyll::Contentful::Serializer.new(entries, config).serialize
```

Comment:
This line is repeated in both export methods, therefore would be nice if it's moved to `#run` and pass the result as a parameter